### ISSUE TITLE
Improve CL formatting

### ISSUE THREAD

**Issue by christineetaylor** [christineetaylor/py-password-toy#6]
Create visual clarity for initial user input, error messages, and generated password.

**Comment by iamshivandev** [comment]
Hello, @christineetaylor, can I work on this issue?

**Comment by christineetaylor** [comment]
Yes, @iamshivandev! This is an excellent first issue (feel free to delay the PR until Hacktoberfest).
